src/store/actions/set_sub_kpis.js

```diff
@@ -0,0 +1,32 @@
+import { firestoreAction } from 'vuexfire';
+import { db } from '@/config/firebaseConfig';
+
+export default firestoreAction(async ({ bindFirestoreRef, unbindFirestoreRef }, departmentId) => {
+  if (!departmentId) {
+    return unbindFirestoreRef('subKpis');
+  }
+
+  const department = db.doc(`departments/${departmentId}`);
+  const departmentExists = await department.get().then(ss => ss.exists);
+
+  if (!departmentExists) {
+    return unbindFirestoreRef('subKpis');
+  }
+
+  const products = await db
+        .collection('products')
+        .where('department', '==', department)
+        .limit(10)
```

Comment:
Er limit(10) der for å hindre at "where('parent', 'in', products)" ikke skal feile? Kan det skje at en avdeling har mer enn 10 produkter? UKE er oppe i 6 f.eks.

Reply:
Ja, det er derfor.

Det kan sikkert skje, og da vil de ikke vises sånn det er nå. Jeg fant bare ikke på noen annen måte å gjøre det på. :sweat_smile: